packages/@apphosting/adapter-nuxt/src/interfaces.ts

```diff
@@ -0,0 +1,13 @@
+/**
+ * Options to help generate output directory
+ */
+export interface OutputBundleOptions {
+  bundleYamlPath: string;
+  outputDirectory: string;
+  wantsBackend: boolean;
+  clientDirectory: string;
+  serverDirectory: string;
+
+  // TODO: review, could not find yet
```

Comment:
We might want it for later to support App Hosting use cases but for now this is not needed. For future reference though, does nuxt have a server we can call `node ./server-file` on?

Reply:
added `server/index.mjs` per the docs: https://nuxt.com/docs/getting-started/deployment#entry-point, although it didn't work in the test app I have for some reason, might be an issue with the app I'm using